tests/selenium/src/actions/actionCheckFileExist.mjs

```diff
@@ -1,8 +1,9 @@
 import { waitBy } from '../lib.mjs';
 import { By } from 'selenium-webdriver';
 import actionIdleWait from './actionIdleWait.mjs';
+import { TIMEOUT_OPTIONAL } from '../options.mjs';
 
-export default async function test({ id = '' }) {
+export default async function test({ id = '', optional = true, timeout = TIMEOUT_OPTIONAL }) {
```

Comment:
Think about waitBy(..., { idle: true }) instead of whole action `actionCheckFileExist()` which is not CheckFileExists but just `CheckIdExists` actually...